science/gromacs/Portfile

```diff
@@ -30,30 +29,32 @@ platforms           darwin
 homepage            http://www.gromacs.org/
 master_sites        http://ftp.gromacs.org/pub/gromacs
```

Comment:
can we change both `homepage` and `master_sites` to use `https`?

Reply:
Done